operations/boot_orchestration/BOS_API_Versions.md

```diff
@@ -32,14 +32,14 @@ BOS v1 versions of the session templates will also remain with `_v1_deprecated`
 
 ## Mechanical differences
 
-When a session is created in BOS v1, BOS starts a Kubernetes job called the Boot Orchestration Agent \(BOA\), which manages all of the nodes.
-BOA moves through one phase at a time, ensuring that all nodes have completed the phase before moving on to the next. This means that all nodes proceed in lock step.
+When a session was created in BOS v1, BOS started a Kubernetes job called the Boot Orchestration Agent \(BOA\), which managed all of the nodes.
```

Comment:
BOA is unique to v1; all references to BOA are no longer relevant.

Reply:
This entire page is outlining the difference between v1 and v2. I felt this should stay in because this is the first release where users are forced to change over to BOS v2. I planned on removing that content in the next release, when all users would have already been forced to make that change.